src/ImageMetadata.jl

```diff
@@ -32,15 +32,15 @@ export
 Construct an image with `ImageMeta(A, props)` (for a properties dictionary
 `props`), or with `ImageMeta(A, prop1=val1, prop2=val2, ...)`.
 """
-mutable struct ImageMeta{T,N,A<:AbstractArray,P<:AbstractDict{String,Any}} <: AbstractArray{T,N}
+mutable struct ImageMeta{T,N,A<:AbstractArray,P<:AbstractDict{String}} <: AbstractArray{T,N}
     data::A
     properties::P
 
     function ImageMeta{T,N,A,P}(data::AbstractArray, properties::P) where {T,N,A,P}
         new{T,N,A,P}(data, properties)
     end
 end
-ImageMeta(data::AbstractArray{T,N}, props::AbstractDict{String,Any}) where {T,N} = ImageMeta{T,N,typeof(data),typeof(props)}(data,props)
+ImageMeta(data::AbstractArray{T,N}, props::AbstractDict{S}) where {T,N,S<:AbstractString} = ImageMeta{T,N,typeof(data),typeof(props)}(data,props)
```

Comment:
I'd use
```julia
ImageMeta(data::AbstractArray, props::Dict{<:AbstractString}) = ImageMeta(data, convert(Dict{String,Any}, props))
ImageMeta(data::AbstractArray, props::AbstractDict) = throw(ArgumentError("properties must be an AbstractDict{String,Any}"))
```

Reply:
Ok, sounds good. Breaking change, but pretty easy to fix.